src/amazon-pay-preview.js

```diff
@@ -0,0 +1,2 @@
+export const amazonPayImage =
```

Comment:
Could we use a regular png file instead? It'd be easier to update it in the future.